### ISSUE TITLE
install django

### ISSUE THREAD

**Issue by lcabrini** [borderitsolutions/amadaa#1]

**Comment by lcabrini** [comment]
OK, django setup is done. Try it out with
`./manage.py runserver`

Then go to the browser and open http://localhost:8000

This should give a default django page. If it does, we are fine so far.

**Comment by BorderITS** [comment]
django is installed

**Comment by lcabrini** [comment]
So we have a confirmation that it works. Can I move to completed or do we want to test it on a Windows machine as well?

**Comment by BorderITS** [comment]
yes i will go for that we need it to work on windows before we move it

**Comment by lcabrini** [comment]
Cool, then I will hold on until we get a confirmation from a Windows user.

**Comment by BorderITS** [comment]
Yes

**Comment by BorderITS** [comment]
Django work's fine on windows now .. I propose we move to Completed

**Comment by lcabrini** [comment]
Yes, I agree.

**Comment by BorderITS** [comment]
ok I'm moving it
